pkg/kubelet/volumemanager/populator/desired_state_of_world_populator_test.go

```diff
@@ -45,12 +45,14 @@ import (
 
 func TestFindAndAddNewPods_FindAndRemoveDeletedPods(t *testing.T) {
 	// create dswp
+	mode := v1.PersistentVolumeFilesystem
 	pv := &v1.PersistentVolume{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: "dswp-test-volume-name",
 		},
 		Spec: v1.PersistentVolumeSpec{
-			ClaimRef: &v1.ObjectReference{Namespace: "ns", Name: "file-bound"},
+			ClaimRef:   &v1.ObjectReference{Namespace: "ns", Name: "file-bound"},
+			VolumeMode: &mode,
```

Comment:
why was this required? same question applies for the other tests where we started adding non-nil volume modes (volume_manager_test.go, fc_test.go, etc)

Reply:
still outstanding